cmd/wclayer/volumemountutils.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+// Simple wrappers around SetVolumeMountPoint and DeleteVolumeMountPoint
+
+import (
+	"path/filepath"
+	"strings"
+	"syscall"
```

Comment:
The `syscall` package is deprecated. We use a script to generate the wrappers around system calls. Please take a look at this file: `https://github.com/microsoft/hcsshim/blob/master/internal/wclayer/wclayer.go` as an example. Add the definitions of the functions that you need (`SetVolumeMountPoint` and `DeleteVolumeMountPoint`) in a new file, adjust the `go generate` command to include the name of that file and then run `go generate` in this folder. That should automatically generate a file named `zsyscall_windows.go` which will contain the wrappers for system calls.

Reply:
In this case it looks like `syscall` is just being used for `UTF16PtrFromString`, which could be used from `golang.org/x/sys/windows` instead.

Reply:
Yeah, I think VSCode did that for me. Easy to fix.